src/test/clientApp/src/main/java/org/lfenergy/operatorfabric/client/common/CreationReport.java

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+package org.lfenergy.operatorfabric.client.common;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class CreationReport {
```

Comment:
There's already a CardCreationReport representing the same object that's generated in the client library (albeit not with the same Lombok annotations), shouldn't we use it instead?

Reply:
Done